README.md

```diff
@@ -197,10 +197,17 @@ yarn run pilet publish --no-fresh --url "https://frontends.service.eu.axinom.net
 ```
 
 Please note, that you need to provide the 'Base64 Encoded Credentials' of a
-service account that has the `Pilets: Publish` permissions as `api-key`. This
+[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) 
+that has the `Pilets: Publish` permissions under the "Micro Frontend Service" as `api-key`. This
```

Comment:
slightly change to something like this?

-----

Please note, that you need to provide the _Base64 Encoded Credentials_ of a
[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) as the value of `api-key`. This service account must contain the `Pilets: Publish` permission of the "Micro Frontend Service" in order to be authorized to perform the action of publishing the workflows.

Reply:
Fixed